core/src/main/java/org/opensearch/sql/ast/expression/Argument.java

```diff
@@ -25,7 +25,7 @@ public class Argument extends UnresolvedExpression {
   //    private final DataType valueType;
   @Override
   public List<UnresolvedExpression> getChild() {
-    return Arrays.asList(value);
+    return Collections.singletonList(value);
```

Comment:
can u use static import for the `Collections.singletonList` method ?

Reply:
Yup. Updated. See comment here: https://github.com/opensearch-project/sql/pull/3193#discussion_r1872382283

Reply:
As suggested by @acarbonetto, I've updated these to use `List.of` in cases where `List` is already imported; otherwise, uses static import of `singletonList`.